code/script.js

```diff
@@ -42,46 +41,47 @@ const sunsetIcon = createWeatherImg("./assets/sunset-icon.png", "sunset");
 
 
 function createFiveDayForecast(filteredForecast) {
+    weatherContainer.innerHTML = ""
```

Comment:
Clearing the HTML for the five days as the first thing before rewriting it (it was in the event listener before)